mean/modules/survey/views/p18_30.html

```diff
@@ -0,0 +1,478 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
again I was just generating pages to help get this done, I thought all of this was part of the standard includes for the pages